### DISCUSSION TITLE
How can I add my own custom css rules?

### DISCUSSION THREAD

**Discussion by cristinacorghi** [gregsullivan/_tw#65] [Q&A] [answered]
I created, following the documentation, a theme with _tw, added it to the wp-content/themes folder and would now like to add my custom rules, i.e. my own fonts and colours. How do I do this?

**Comment by gregsullivan** [comment, marked as answer]
Hi Cristina—thanks for posting your issue!

In terms of getting fonts and colours into Tailwind, the process is the same as described in these two documents:

https://tailwindcss.com/docs/font-family#customizing-your-theme
https://tailwindcss.com/docs/customizing-colors#using-custom-colors

You would make those changes in your `./tailwind/tailwind.config.js` file.

If you're looking at add your own self-hosted fonts, please see here:

https://underscoretw.com/docs/development/#custom-css

You'll find a `fonts.css` file for declaring self-hosted web fonts should you need to do so.

Please let me know if you have any other questions!

**Reply by cristinacorghi** [reply]
Hi Greg,
Now it works!
Thank you so much!

**Reply by gregsullivan** [reply]
You're very welcome—happy to hear it!

Please don't hesitate to post again if you have any other questions.
